### PR TITLE
Adrv9009 support

### PR DESCRIPTION
**Pull request by mhennerich**

### CODE REVIEW

File: plugins/adrv9009.c [hunk 1]
```diff
@@ -0,0 +1,1401 @@
...
+	/* The gain_control_mode iio attribute should be set prior to setting
+	 * hardwaregain iio attribute. This is neccessary due to the fact that
+	 * some control modes change the hardwaregain automatically. */
+	ch = iio_device_find_channel(dev, "voltage0", false);
+	value = read_token_from_ini(ini_fn, THIS_DRIVER,
+	                            PHY_DEVICE".in_voltage0_gain_control_mode");
+
+	if (ch && value) {
+		iio_channel_attr_write(ch, "gain_control_mode", value);
+		free(value);
```
[the remark this answers is not in the document]
Comment (mhennerich) [reply plugins/adrv9009.c]: Error checking is a bit lax. The plugin initially checks the presence of all required devices, then it assumes that also all channels are present. I'll add a few more checks to adrv9009_init().

**Comment by mhennerich** [comment]
force pushed a revised version

**Comment by commodo** [comment]
Will merge later today if no objections

**Comment by commodo** [comment]
Update: maybe let's push it for tomorrow [the merge].
It might give more time for others to take a look.

**Pull request merged**